### ISSUE TITLE
Syntax highlight does not refresh after removing a block comment /* */

### ISSUE THREAD

**Issue by rlalance** [Open-Turing-Project/OpenTuring#34]

**Comment by ghost** [comment]
The old editor is deprecated. Use the new editor instead. There's too many problems with the old one to bother trying to fix it.
https://github.com/Open-Turing-Project/turing-editor-qt

**Comment by rlalance** [comment]
Will try. I already have the new editor though.

**Comment by rlalance** [comment]
Indeed, already have the new editor. syntax colouring does not update when using /\* */
